package.json

```diff
@@ -33,11 +34,12 @@
     "typescript": "3.0.1"
   },
   "devDependencies": {
-    "@nestjs/testing": "5.1.0",
+    "@nestjs/testing": "^5.7.4",
```

Comment:
Could you please change to "@nestjs/testing": "5.7.4".
Use this symbol "^" could generate problems with versions of libraries to the team because the symbol get the latest version.

Reply:
Understood!